### DISCUSSION TITLE
Please help to deploy SSL certs on 2.10.4

### DISCUSSION THREAD

**Discussion by barmalej1** [netbox-community/netbox-docker#435] [Getting Help] [answered]
Previously had them mapped to Nginx container. I understand that Unit supports certificates, but their guides are not very clear

**Comment by cimnine** [comment, marked as answer]
Please see https://github.com/netbox-community/netbox-docker/wiki/TLS for the suggested TLS setup.

**Reply by ryanmerolle** [reply]
I really cannot stress how easy this process is with hitch as documented in the wiki
